### PR TITLE
merge deepks into develop

### PR DESCRIPTION
**Pull request by maki49**

### CODE REVIEW

File: source/module_base/global_variable.cpp [hunk 1]
```diff
@@ -118,8 +118,8 @@ std::string	global_atom_card = "STRU";
 std::string	global_kpoint_card = "KPT";
 std::string	global_wannier_card;
 
-std::string	global_pseudo_dir = "./";
-std::string global_orbital_dir = "./";   // liuyu add 2021-08-14
+std::string	global_pseudo_dir = "";
```
Comment (dyzheng) [inline source/module_base/global_variable.cpp:121]: Why do you modify this default from "./" to ""? Did any problem about that?

File: source/src_lcao/ELEC_scf.cpp [hunk 2]
```diff
@@ -552,9 +552,17 @@ void ELEC_scf::scf(const int &istep)
 					GlobalV::ofs_running << " " << GlobalV::global_out_dir << " final etot is " << GlobalC::en.etot * ModuleBase::Ry_to_eV << " eV" << std::endl;
 				}
 #ifdef __DEEPKS
-				if (INPUT.deepks_scf)	//caoyu add 2021-06-04
+				if (INPUT.out_descriptor)	//caoyu add 2021-06-04
```
Comment (dyzheng) [inline source/src_lcao/ELEC_scf.cpp]: Please avoid to use "INPUT.", you can add a new variable in global_variable.h and use it for replace that.

File: source/src_lcao/FORCE_gamma.cpp [hunk 3]
```diff
@@ -75,9 +75,9 @@ void Force_LCAO_gamma::ftable_gamma (
     if (INPUT.deepks_scf)
```
Comment (dyzheng) [inline source/src_lcao/FORCE_gamma.cpp]: Don't use "INPUT." here, using of global variant is better now.

File: source/module_base/global_variable.cpp (hunk 1 above)
Comment (maki49) [reply source/module_base/global_variable.cpp:121]: because deepks-kit will convert file-dir to absolute path, it can't work if begin with './'

**Pull request merged**